.github/workflows/release-node.yml

```diff
@@ -0,0 +1,97 @@
+name: Release node image
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: 'Tag for the images (e.g.: beta)'
+        required: true
+      apify_version:
+        description: 'Apify SDK version (e.g.: ^1.0.0)'
+        required: true
+      node_version:
+        description: 'NodeJs version for image actor-node'
+        required: false
+        default: '14'
+
+  repository_dispatch:
+    types: [ build-images ]
+
+env:
+  RELEASE_TAG: ${{ github.event.inputs.release_tag || github.event.client_payload.release_tag }}
+  APIFY_VERSION: ${{ github.event.inputs.apify_version || github.event.client_payload.apify_version }}
+  NODE_VERSION: ${{ github.event.client_payload.node_version || github.event.inputs.node_version }}
+
+jobs:
+  # Build master images that are not dependent on existing builds.
+  build-main:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image-name: [node]
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Set Dependency Versions
+        run: |
+          cd ${{ matrix.image-name }}
+          node ../.github/scripts/set-dependency-versions.js
+      -
+        # It seems that it takes at least two minutes before a newly published version
+        # becomes available in the NPM registry. We wait before starting the image builds.
+        name: Wait For Package Registry
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 2 # timeout for a single attempt
+          max_attempts: 3
+          retry_wait_seconds: 60 # wait between retries
+          command: cd ${{ matrix.image-name }} && npm i --dry-run
+      - 
+        name: Build image with node version.
+        uses: docker/build-push-action@v2
+        if: ${{ env.RELEASE_TAG == 'latest' }}
+        with:
+          context: ./${{ matrix.image-name }}
+          file: ./${{ matrix.image-name }}/Dockerfile
+          build-args: VERSION=${{ env.NODE_VERSION }}
```

Comment:
I would name the arg `NODE_VERSION` too. This way, if we introduce another version, we'll have to rename it.